jsapp/js/components/submissions/audioCell.tsx

```diff
@@ -31,10 +32,20 @@ export default function AudioCell(props: AudioCellProps) {
         </span>
       )}
 
-      {typeof props.mediaAttachment === 'object' &&
-        props.mediaAttachment?.download_url && (
-          <MiniAudioPlayer mediaURL={props.mediaAttachment?.download_url} />
-        )}
+      {
+        typeof props.mediaAttachment === 'object' &&
+        props.mediaAttachment?.download_url &&
+        !props.mediaAttachment?.is_deleted &&
+      (
+        <MiniAudioPlayer mediaURL={props.mediaAttachment?.download_url} />
+      )}
+
+      {
+        typeof props.mediaAttachment === 'object' &&
+        props.mediaAttachment?.is_deleted &&
+      (
+        <DeletedAttachment />
+      )}
 
       <Button
```

Comment:
Should we wrap this in a check for is_deleted, too? At the moment, the button still shows up when an attachment is deleted.

Reply:
So we want all the processing work to be accessible even if user deletes attachment